hathor/feature_activation/bit_signaling_service.py

```diff
@@ -58,6 +58,9 @@ def start(self) -> None:
         Log information related to bit signaling. Must be called after the storage is ready and migrations have
         been applied.
         """
+        if not self._feature_service.is_usage_enabled():
```

Comment:
I feel it should be done somewhere else. It seems error prone to call `start()` and get no exception because it just skipped initialization.

Reply:
This has been removed from this PR.